.drone.local.yml

```diff
@@ -20,9 +20,10 @@ pipeline:
       VIC_ESX_TEST_URL: ${VIC_ESX_TEST_URL}
       LOG_TEMP_DIR: install-logs
       DRONE_SERVER:  ${DRONE_SERVER}
+      DRONE_MACHINE: ${DRONE_MACHINE}
```

Comment:
this might not be set currently right?

Reply:
No. For local builds, we don't volume mount env var file, so instead set them as env var. It is needed.